src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -141,7 +141,8 @@ public Object instanceMainPostParseArgs() {
                         " " + System.getProperty("os.arch") + "; " + System.getProperty("java.vm.name") +
                         " " + System.getProperty("java.runtime.version") +
                         "; Version: " + commandLineParser.getVersion() +
-                        " " + (Defaults.USE_ASYNC_IO ? "asyncIO": "syncIO") +
+                        " " + (Defaults.USE_ASYNC_IO_FOR_SAMTOOLS ? "asyncIOBam": "syncIOBam") +
+                        " " + (Defaults.USE_ASYNC_IO_FOR_TRIBBLE ? "asyncIOTribble": "syncIOTribble") +
```

Comment:
This shouldn't be here in the main program name / version String. Recommend instead outputting a `logger.info()` message with the async settings, which is more consistent with the way we communicate other engine-level settings in GATK.